modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Detail.js

```diff
@@ -26,25 +27,24 @@ export default function Detail({
 }) {
 	return (
 		<>
-			<div className="d-flex p-2">
-				<ClayButtonWithIcon
-					className="text-secondary"
+			<div className="d-flex pb-3 pt-1">
+				<ClayButton
```

Comment:
nit: You could use `ClayButtonWithIcon` if you don't want to compose it yourself.

Reply:
I did it in this way because `<ClayButtonWithIcon` adds the class `btn-mnospaced` by default and I need it not to be here.

It could be an improvement to the component in Clay? Add support to pass `monospaced` prop?

Reply:
@carloslancha yeah we can probably add that as a prop and default it to true, that way you can disable it and we won't have breaking changes. Ill create an issue

Reply:
nice, thx!